content/en/docs/home/contribute/create-pull-request.md

```diff
@@ -47,11 +47,24 @@ that is the best fit for your content.
 
 ## Viewing your changes locally
 
-When you submit a pull request, you can see a preview of your changes at
-[Netlify](https://www.netlify.com/). If you prefer to see a preview of your changes
-before you submit a pull request, you can build a preview locally. For more information, see
-[Staging locally](/docs/home/contribute/stage-documentation-changes/#staging-locally-without-docker). 
+You can use Hugo to see a preview of your changes locally.
+
+1. [Install Hugo](https://gohugo.io/getting-started/installing/).
```

Comment:
Should we mention the version limitations that Melissa found?

Reply:
Done. Added "version 0.40.3 or later".